### PR TITLE
Update to v0.30.1 (RLN v2)

### PR DESCRIPTION
**Pull request by Ivansete-status**
* change the nwaku image from v0.29.0 to v0.30.0
* change the smart contract address
* Give details about Polygon zkEVM in README.md

### CODE REVIEW

**Review by gabrielmer — APPROVED** [review]
Looks amazing! Thanks! 😍

**Review by rymnc — APPROVED** [review]
LGTM

**Review by alrevuelta — APPROVED** [review]

**Comment by Ivansete-status** [comment]
Thanks for the reviews! I won't merge it until we deploy the `v0.30.0` in `waku.sandbox`.

@rymnc , @alrevuelta - can confirm that the `waku.test` behaves well from your PoV?

File: README.md [hunk 1]
```diff
@@ -10,10 +10,21 @@ Ready to use docker-compose to run your own [nwaku](https://github.com/waku-org/
 **📝 0. Prerequisites**
 
 You need:
-* Ethereum Sepolia WebSocket endpoint. Get one free from [Infura](https://www.infura.io/).
-* Ethereum Sepolia account with some balance <0.01 Eth. Get some [here](https://www.infura.io/faucet/sepolia).
+* Polygon zkEVM Cardona https endpoint. Get one free from [Alchemy](https://www.alchemy.com/).
+* Polygon zkEVM Cardona (test) account with some ETH balance (<0.1 ETH). Get some [here](https://faucet.polygon.technology/).
```
Comment (rymnc) [inline README.md:14]: you can also add https://faucet-cardona.rymnc.com/ as the recommended faucet

**Review by NagyZoltanPeter — APPROVED** [review]
LGTM!
Btw is it possible that one can lower or raise the user limit?

**Pull request merged**